postgres/README.md

```diff
@@ -119,6 +122,30 @@ PostgreSQL default logging is to `stderr` and logs do not include detailed infor
       #log_destination = 'eventlog'
     ```
 
+2. To gather detailed duration metrics and make them searchable in the Datadog interface, they will need to be 
+   configured inline with the statement themselves.  See below for the recommended config, note that counterintuitively 
+   both `log_statement` and `log_duration` options are commented out.
+   
+   This config will log all statements, but to optionally reduce the output to those which have a certain duration,
+   set the `log_min_duration_statement` value to the desired minimum ms:
+
+    ```conf
+      logging_collector = on
+      log_directory = 'pg_log'  # directory where log files are written,
+                                # can be absolute or relative to PGDATA
+      log_filename = 'pg.log'   #log file name, can include pattern
+      log_min_duration_statement = 0    # -1 is disabled, 0 logs all statements
+                                        # and their durations, > 0 logs only
+                                        # statements running at least this number
+                                        # of milliseconds
+      #log_statement = 'all'     #log all queries
+      log_line_prefix= '%m [%p] %d %a %u %h %c '
+      log_file_mode = 0644
+      ## For Windows
+      #log_destination = 'eventlog'
```

Comment:
I see this snippet duplicates options from step 1) that aren't immediately relevant to collecting duration metrics, e.g. `log_filename`, `logging_collector`, etc. Should we switch this into "To gather detailed duration metrics, comment out `log_statement` and add the `log_min_duration_statement`"?

Reply:
Also maybe if there's official PostgreSQL docs on this, we can probably link to it for users to find more info by themselves (e.g. one relevant question might be "why do I need to comment out `log_statement`?").

Reply:
Good point - updated the preceding paragraph.